### DISCUSSION TITLE
Problem installing homebrew on a M1 mac

### DISCUSSION THREAD

**Discussion by aviadbus** [Homebrew/discussions#4057] [Getting started] [answered]
Hi, I am trying to install home brew using
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
on a new mac (m1)
and get this error:
`==> Downloading and installing Homebrew...
HEAD is now at 7d04dd9d6 Merge pull request #14256 from eugenesvk/patch-2
error: open("Formula/grammarly-languageserver.rb"): Operation not permitted
fatal: Unable to process path Formula/grammarly-languageserver.rb
Error: Could not 'git stash' in /opt/homebrew/Library/Taps/homebrew/homebrew-core!
Please stash/commit manually if you need to keep your changes or, if not, run:
  cd /opt/homebrew/Library/Taps/homebrew/homebrew-core
  git reset --hard origin/master
Failed during: /opt/homebrew/bin/brew update --force --quiet
`

I tried the suggested commands and it did not help. 
Would be grateful for any assistance 
Thanks!

**Comment by p-linnane** [comment, marked as answer]
Try `rm -rf /opt/homebrew/` and then run the installer again.

**Reply by aviadbus** [reply]
Thanks!!! solved.
